src/Opdex.Platform.WebApi/OpenApi/Wallets/RefreshAddressBalanceOperationProcessor.cs

```diff
@@ -15,7 +15,7 @@ public bool Process(OperationProcessorContext context)
 
         var tokenAddressParameter = context.OperationDescription.Operation.Parameters.First(parameter => parameter.Name == "token");
         tokenAddressParameter.Schema.Example = "tF83sdXXt2nTkL7UyEYDVFMK4jTuYMbmR3";
-        tokenAddressParameter.Schema.MinLength = 30;
+        tokenAddressParameter.Schema.MinLength = 3;
         tokenAddressParameter.Schema.MaxLength = 42;
```

Comment:
For this Refresh address balances can keep 30 min length I think, we won't be refreshing CRS balances unless we start to persist them.